app/Models/Employer.php

```diff
@@ -8,7 +8,7 @@
 use Illuminate\Notifications\Notifiable;
 use Laravel\Sanctum\HasApiTokens;
 
-class User extends Authenticatable
+class Employer extends Authenticatable
```

Comment:
Perhaps you could have also left this model untouched and related it to another one for the Employers, on one on one relationship, up to you